zipline/assets/assets.py

```diff
@@ -259,6 +259,10 @@ def _retrieve_futures_contract(self, sid):
                 data['expiration_date'] = pd.Timestamp(
                     data['expiration_date'], tz='UTC')
 
+            if data['auto_close_date']:
+                data['auto_close_date'] = pd.Timestamp(
+                    data['auto_close_date'], tz='UTC')
+
```

Comment:
I like that, and it fits well parallel to the _convert_asset_str that we already do. Adding now.